docs/next/guides/rows/row-virtualization.md

```diff
@@ -27,7 +27,9 @@ Note that the data grid without virtualization enabled will only work with relat
 
 You can experiment with the `viewportRowsRenderingOffset` config option, which determines the number of rows displayed outside the visible viewport. If the number passed to that option is greater than the total columns in your data set, then the virtualization will be practically turned off.
 
-::: warning Proceed with caution, as it will affect the overall performance of the grid. :::
+::: warning
```

Comment:
Can you do the same for `docs/9.0/guides/rows/row-virtualization.md`? I think that 9.0 would stay there till release so it's important to sync both versions.